app/api/services/evidence.py

```diff
@@ -83,6 +84,112 @@ def get_previous_submitted_evidence_for_supplier_and_domain(self, evidence_id, d
         evidence = query.first()
         return evidence
 
+    def get_data(self, evidence_id):
+        category_name_max_daily_rate = (
+            db.session.query(
+                Domain.name.label('category'),
+                Evidence.data['maxDailyRate'].label('maxDailyRate')
+            )
+            .join(Evidence, Evidence.domain_id == Domain.id)
+            .filter(Evidence.id == evidence_id)
+        )
+
+        domain_criteria_id = (
+            db.session.query(
+                func.json_array_elements_text(Evidence.data['criteria']).label('dc_id')
+            )
+            .filter(Evidence.id == evidence_id)
+            .subquery()
+        )
+
+        result = (
+            db.session.query(
+                domain_criteria_id.c.dc_id,
+                DomainCriteria.name.label('domain_criteria_name'),
+                Evidence.data['evidence'][domain_criteria_id.c.dc_id].label('evidence_data'),
+            )
+            .join(DomainCriteria, DomainCriteria.id == domain_criteria_id.c.dc_id.cast(Integer))
+            .filter(Evidence.id == evidence_id)
+        )
+        evidence = {}
+        evidence_data = [evidence_values._asdict() for evidence_values in result.all()]
+        evidence['evidence'] = evidence_data
+
+        category_max_rate = {}
+        for c_values in category_name_max_daily_rate.all():
+            category_max_rate = c_values._asdict()
+
+        evidence['category'] = category_max_rate.get("category")
+        evidence['maxDailyRate'] = category_max_rate.get("maxDailyRate")
+
+        return evidence
+
+    def get_evidence_data(self, evidence_id):
```

Comment:
This doesn't look like it's used anywhere.  Is this a work in progress or can it be deleted?

Reply:
removed